src/pages/EntityDetails/EntityDetails.test.tsx

```diff
@@ -86,33 +87,53 @@ describe("Entity Details Container", () => {
   });
 
   it("should display the correct window title", () => {
-    renderComponent(<EntityDetails />, { path, url, state });
+    renderComponent(<EntityDetails modelWatcherError={null} />, {
```

Comment:
If `modelWatcherError` is set to an optional prop then all these tests won't need to be updated.

Reply:
Done.